docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,162 @@
+.. _acsincome-data:
```

Comment:
Thanks for your PR! However, you need to remove all these other files from the other PR. You can do that manually or by running `git rebase -i HEAD~10` (or replace 10 with whatever number of commits you want to drop) and then write "drop" for the corresponding commits. Whatever suits you better.